molgenis/molgenis-emx2
# fix: Description from schema removed after updating settings

fixes #3721 

What are the main changes you did:
- Copy constructor was missing the description field 

how to test:
- Go to https://preview-emx2-pr-3967.dev.molgenis.org/apps/central/#/
- Create a schema with description
- Go to settings > advandced settings, and create a new setting
- See that the description is still available at https://preview-emx2-pr-3967.dev.molgenis.org/
todo:
